tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -344,20 +356,23 @@ void TRTEngineOp::ComputeAsync(tensorflow::OpKernelContext* ctx,
       std::vector<int> trt_shape(dims.nbDims + 1);
       trt_shape[0] = num_batch;
       for (int j = 0; j < dims.nbDims; j++) trt_shape[j + 1] = dims.d[j];
-      OP_REQUIRES_OK(
-          ctx, TensorShapeUtils::MakeShape(trt_shape.data(), trt_shape.size(),
-                                           &output_shape));
+      auto status = TensorShapeUtils::MakeShape(
```

Comment:
Could this ever fail?

Reply:
I'm not 100% sure. If it will fail, it must be also fail in native execution unless there is a bug in the conversion. In case of bugs we'd better retry.

Reply:
In case of the errors in conversion, networks should be unusable. There is no point in trying and failing all the time. Effectively this will slow down the execution of the network.